### DISCUSSION TITLE
Agent Roles (printer) (again)

### DISCUSSION THREAD

**Discussion by krgomez** [ArctosDB/arctos#5404] [Ideas]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Goal**
To enhance Arctos for use by non-biological collections. See #2478.

**Context**
Overview of the challenge:
There are different roles that fall under the category of “creator” that are important for us to distinguish between. In many cases in our collection multiple people have played different roles in the creation of a work and we need a way to associate both the agent and their role with the object record. [Creator Role is a CDWA core category](http://www.getty.edu/research/publications/electronic_publications/cdwa/14creation.html#Creator-Role) that is described as the role or activity performed by a creator or maker in the conception, design, or production of a work. Examples include: artist, painter, designer, draftsman, engraver, muralist, potter, modeller, sculptor, goldsmith, publisher, architectural firm.

Challenge for Fine Arts collection:
There are a number of prints and photographs in the UAM:Art collection where the printer is different from the artist. We currently have no way of distinguishing these roles in the creation of a work, however it is important to do so. As an example, we have a number of Inuit prints in the collection where the artist, printer, as well as sometimes other roles in the creative process, are distinguished and important to the works. We also have a number of photographs where the artist is not the printer and where we know who is, and it would also be relevant to record these printers as agents in the catalog records. Currently printers are either not listed at all or sometimes listed as a “maker” which can lead to misconceptions about creative authorship and rights.

Other collections that may have this issue:
Archives? Ethnology and History? I’m thinking things like composer, author, etc., would all be beneficial sub-categories under “creator”.

Possible solution:
Would it be possible to have a broad agent role “creator” (see #2578) and also a way of clarifying that creator role to something specific like “printer”, “artist”, “composer”, “publisher” etc.?

**Comment by Jegelewicz** [comment]
Would this be related to collector roles? https://arctos.database.museum/info/ctDocumentation.cfm?table=CTCOLLECTOR_ROLE?

**Comment by krgomez** [comment]
Yes. Another idea is to add printer as an additional role in the COLLECTOR_ROLE table. We were thinking that because it is technically a sub-role of creator/maker that maybe it would be more ideal to find a way to be able to distinguish between multiple creator/maker roles in addition to printer.

**Comment by AJLinn** [comment]
We deal with various "sub-roles" of creators in EH also, when composite items are made by multiple people (e.g., dolls with heads carved by one person and clothing made by another; clothing assembled by one, decorations embroidered by another; etc.). We typically make this distinction in the remarks field. 

I can imagine how this might be a useful field to bring greater specificity to the creator roles in our complex items, though the various kinds of sub-roles for art might be more standardized than for EH collections. I would guess I would continue to describe the specific tasks and specific elements undertaken by the various creators in the remarks field. Maybe there could be an agent remarks field that we use to specifically comment on the agent role so folks don't have to go down to the overall remarks to see that info?

**Comment by dustymc** [comment]
Why not catalog the derivative separately?

**Comment by Jegelewicz** [comment]
> Why not catalog the derivative separately?

Because it's a doll, not a "doll head" and "doll clothing". While I could see how cataloging them separately makes data sense, it doesn't make object sense. However, I am also not a fan of adding too many too-specific roles either and "printer" feels like that to me. We will eventually end up with "painter", "carver", and on down the line.

> describe the specific tasks and specific elements undertaken by the various creators in the remarks field. Maybe there could be an agent remarks field that we use to specifically comment on the agent role so folks don't have to go down to the overall remarks to see that info?

I like this - we have comments or remarks in every other section of the record, why not here as well? "Agent Remarks" seems like a better way to provide that information instead of it being buried in "Remarks" along with a bunch of other stuff.

**Comment by dustymc** [comment]
Maybe I'm not understanding something, but I think a print is a derivative, not ~part of an original. It's a different THING, but still related. Maybe closer to a clone than offspring, to relate to biology-land.

I have no particular objections to "collector remarks" - needs a new issue though.

**Comment by Jegelewicz** [comment]
But what if all you have is the print.....

The actual art - say a woodblock was done by Artist
The print was made by - Printer

**Comment by krgomez** [comment]
Here's an example, where Victoria Mamnguqsualuk is the artist and Irene Avaalaaqiaq Tiktaalaaq is the printer. I have added them both as makers, but you can see how it would be preferable to clearly specify that Tiktaalaaq is the printer for this stonecut print. I've added printers like this for at least some of the Inuit prints in the collection, but for the photography collection I have not been doing that because it would be particularly confusing (and only sometimes is the printer a different agent). I have sometimes just added their names into the remarks, but I feel like that's not ideal.

![Screen Shot 2020-04-03 at 5 23 45 PM](https://user-images.githubusercontent.com/40579515/78415695-e5110780-75cf-11ea-8910-67d76bbcd84a.png)

**Comment by dustymc** [comment]
>But what if all you have is the print.....

What if all you have is the parasite? We deal with separate but closely linked things all the time. There are multiple items of scientific interest so we catalog them all when we need to ask questions of the intersection of data objects.

I don't think that's QUITE what's going on here, but it's not clear to me what the item of scientific interest is either. If I scan this print, print a new "plate" from that, and print a duplicate of this item from that, am I the artist? Printer? Something new? Is my creation a new object? Does it get added to this record? That is, are we building single-dimensional data because nobody actually cares, or will ever plausibly care, about anything deeper, or are we overloading a record for some reason?

I do see the original point of this issue a little more clearly now. From the screenshot I have little idea who did what, and that leads to uncertainty about the rest of the data as well - what exactly happened at the single place-time?

Unrelated, http://arctos.database.museum/agent.cfm?agent_id=21315801&agent_name=Victoria%20Mamnguqsualuk has a giant blob of text at the bottom. Please see https://handbook.arctosdb.org/documentation/agent.html#creating--maintaining-agents

>Use Addresses when possible. A partial address (“Ohio”) or link to a biography (e.g., Wikipedia – these may be made via Address Type “URL”) is very likely to be useful. Avoid using remarks for address information.

**Comment by AJLinn** [comment]
I think that the Inuit prints are a unique situation, where the artist is the individual who creates the design by carving the soapstone plates and then the printer is also important because of the way they apply the paint to the stone plates and undertake the printing process. I'm not sure how common this is in other artist/printer relationships.

I do think by creating an Agent Remarks field we'll be able to clearly distinguish these very specific types of activities by two separate artists. @krgomez do you think this would be an adequate way to distinguish the separate roles? By doing that we're not making a value judgement on whose role is more important, we're just clarifying.

**Comment by krgomez** [comment]
I fixed the agent remarks for Mamnguqsualuk -- thanks for pointing out that I shouldn't be adding URLs to agent remarks. I have now added them as URL addresses. There are some others where I have done that and I will fix them as I come across them.

It seems to me that an Agent Remarks field would likely solve the problem. Would these remarks then appear along with the agent name in the agent boxes in catalog records? Clarity is what we're after.

Like what Angie said, the roles in printmaking can be specific and important to the artworks. Very often in contemporary printmaking the artist is responsible for the design, matrix-making and printing of their blocks/plates/stones/etc, but historically (and in some contemporary situations such as is the case with Inuit printmaking centers such as Cape Dorset in Canada and other professional print shops around the world that employ technicians and master printers to work directly with artists to make, print and then publish their prints) these roles were often distinct and sometimes acknowledged. There is quite a lot of technique involved in creating matrices and printing (either by hand or press) and so in the cases where an engraver/carver, printer or other role is acknowledged as distinct from the artist, it can be important to credit their role in the creation of a print. If you are interested, there is an amazing documentary about the Cape Dorset printmaking tradition called Kinngait: Riding Light into the World that I highly recommend!

> I do see the original point of this issue a little more clearly now. From the screenshot I have little idea who did what, and that leads to uncertainty about the rest of the data as well - what exactly happened at the single place-time?

Yes, sometimes there are layers of time/place/agent roles involved in the making of an artwork. We are currently only recording one manufacture event for each work. However with photographs and prints specifically there could technically be two manufacture/creation events, the creation of the negative or matrix and the printing. Sometimes there can be a lot of time between these two events, even years. We usually only have information about one event for prints but we sometimes know the different agent roles, like I was talking about above. For photographs in the UAM:Art collection we do sometimes have information about both events. Maybe we should be recording two events for those. Basically there is complexity in creation and sometimes multiple individuals/groups/businesses can be involved through different roles beyond "artist", but it is of course very important to maintain clarity about who the artist is.

**Comment by dustymc** [comment]
Great, I'll just hijack this Issue so we have the context.

Yes the remark would appear with the agent-in-role data.

>technically be two manufacture/creation events

You should consider two cataloged items. Are the negative and print two parts of the same THING, or two related THINGs in their own right? I don't have an answer, but I think it's a question worth asking. Citability is probably the biggest consideration from my perspective - given "we looked at {specimen} and ..." in a publication, is there repeatability - can I come along in 100 years later and get the same material, or do I have to guess at the details?

**Comment by dustymc** [comment]
* Add collector.collector_remarks (varchar, PRINT-char)
* Add SpecimenDetail/Agents edit capability
* Display in SpecimenDetail

**Comment by krgomez** [comment]
Another thought on this -- I'm not sure this is the ideal situation for searching by artist name. With remarks used to differentiate between creator (maker) roles, that still leaves us searching by creator, which will lead to confusing results. For example, we have a photograph by Frederick Dean that was printed by Malcolm Lockwood, who is also the artist for other photographs in the collection. So when we search for creator=Malcolm Lockwood, expecting to see all works for which Lockwood is the artist, we would also see this Frederick Dean photograph among the results. Often, these printers (for both photographs & original prints) are artists as well outside of professional work as printers. It goes back to the problem of maintaining clarity of roles/who is the artist of a work.

**Comment by dustymc** [comment]
>maintaining clarity of roles/who is the artist of a work.

I'll go back to my initial response: If they're different THINGS, then why are they cataloged together?

**Comment by krgomez** [comment]
They're not different things. In the cases of both photographs and prints that I keep mentioning, we have a physical photographic print, lithograph, stonecut, woodcut, etching, etc on paper. For any one of these prints we have an artist which we record as the maker. Often the artist is either also the person that did the printing or we just don't know either way. But sometimes we know that the printer is a different agent.

**Comment by Jegelewicz** [comment]
@dustymc it took two people to create the THING. An artist who made some kind of pattern or photograph and a craftsperson who used the pattern or photographic negative to print the work that is being cataloged.

**Comment by dustymc** [comment]
>They're not different things.

Then all of the listed agents are involved in creating the single THING, no?

**Comment by Jegelewicz** [comment]
Yes...BUT they all had SPECIFIC roles in the creation!

**Comment by Jegelewicz** [comment]
Technically, I could argue that the collector and preparator all had a role in "collecting" the specimen, yet we give each of them a special role - this is all @krgomez is asking for.

**Comment by campmlc** [comment]
I agree with Teresa's point.

**Comment by AJLinn** [comment]
Yeah, I've been thinking a bit more about this. I think we should probably create these sub-roles for agent creators. They only show up if you use them so it's not like there's going to be a bunch of empty fields for people not using them. If "creator role" is considered a core category for art collections we should just add it so the art collections can be using the best practices.

I do think we should still create an Agent Remarks in addition to adding these roles. If we have particular explanations we'd like to use to further explain the roles of these agents it's best that they're right up there next to the string of agents & their roles rather than buried down below the attributes.

**Comment by Jegelewicz** [comment]
> I do think we should still create an Agent Remarks in addition to adding these roles. If we have particular explanations we'd like to use to further explain the roles of these agents it's best that they're right up there next to the string of agents & their roles rather than buried down below the attributes.

Agree completely!

**Comment by dustymc** [comment]
>that the collector and preparator all had a role in

I'd have 100% dropped 'preparator' the very day the "mamm" and "bird" and etc. tables became Attributes with individual determiners. Maybe someday, but it's so steeped in "tradition" that it'll probably outlive me.

I'm still not sure ya'll are cataloging the right thing. That's fine, but I do hope you've considered the possibility.

> "creator role" is considered a core category for art collections

Despite the CDWA being a bit less "not a database"-ish than eg DWC, that's a mapping-for-export issue and not something that should drive design or even terminology.

If you are sure (enough) that you're cataloging the right thing and that "printer" is critical in that, then I believe my remaining concerns are twofold:

1. Is this opening the doors to infinite collector roles? Is "third-shift taillight boxer-upper" (or maybe "crankcase sand cast maker" to stick to the "printing from mold" theme!) going to be viewed as necessary for things like http://arctos.database.museum/guid/UAM:EH:UA92-017-0001? If so, is The Community OK with that?

2. How will this be consistently used? What precisely will you do for something that is for whatever reasons considered "the item" on it's own and is produced only by printing, or where the object involves only creating things from which others might be printed, and how will that be consistent with objects where printing is but part of the creation process? I cannot see a consistent path that involves one generic and one specific "creator" role.

**Comment by AJLinn** [comment]
@dustymc I think you make some good points, which drive me to the Dublin Core Metadata Terminology, which is the standard for libraries and archives and the non-biological analog to Darwin Core.
https://www.dublincore.org/specifications/dublin-core/dcmi-terms/

Darwin Core has a few terms associated with agents:
**creator**: An entity primarily responsible for making the resource.
**contributor**: An entity responsible for making contributions to the resource.

What if we making these two our standard agent roles in addition to collector, (ignoring the preparator, which I agree is not super important and has been over-used in some collections) which are then clarified then thru the Agent Remarks?

A bit more info on DCMI - there are 15 terms considered the "Dublin Core Metadata Element Set"
https://en.wikipedia.org/wiki/Dublin_Core

**Comment by krgomez** [comment]
I talked with Mareca this morning and we both like your suggestion, @AJLinn . Contributor is a neutral, interdisciplinary term that could cover a range of roles while leaving creator separate, eliminating the problem of potential misattribution. We can't think of any drawbacks or problems with contributor and it would cover printer which we could clarify in the agent remarks. The term also doesn't rule out an artistic contribution either, which could be useful for some works while maintaining clarity about the artist of a work through the creator role.

**Comment by dustymc** [comment]
I do like "creator" and "contributor" in that they seem less likely to lead to an infinite number of roles. I'm not sure that addresses the potential (inevitability??) of arbitrary usage, however.

* just a "plate"-->person is creator
* just a "print"-->person is ?????????????????????? and person who created the "plate" is also {flip a coin??} (I'd probably call the printer - the person who made the physical thing being cataloged - as the main "creator" and the "plate-maker" the 'contributor' but I think that may be opposite of the original comment and/or how users might view those data.)
* both are cataloged together --> same as above, but with even more ambiguity!

MAYBE that's just a matter of documentation.

I'm not entirely clear on the goals, I think. Are we trying to support precise agent-in-role queries, or to clarify why an agent is listed on a record, or something else? Precision probably requires ~infinite collector roles; that's not necessarily a bad thing, but it is something that we should design rather than finding ourselves unexpectedly submerged in. Remarks probably serves as a clarifier, and maybe does so a bit better under something like creator/contributor. @krgomez can you elaborate on this a bit more?

It also occurs to me that terms like "printer" may become problematic - eg, a "plate" could be 3D printed then "prints" created from that. I think that's just the normal documentation and thinking at scale issues.

**Comment by krgomez** [comment]
> Are we trying to support precise agent-in-role queries, or to clarify why an agent is listed on a record, or something else?

I would say both. We need to be able to correctly catalog intellectual and technical contributions associated with objects. Who created the object is one of the most important pieces of information we have about objects and there could be legal implications if we do not make those relationships clear. It needs to be clear to all users in all contexts (search results, catalog records, etc.) who the artist/creator is and we also need to be able to track other agents that may have contributed to the creation of the work. In thinking about the collection, contributor roles that apply for us include printer, publisher and foundry/casting for sculpture. "Publisher" is actually a core metadata element for Dublin Core in addition to "creator" and "contributor". So maybe that should be considered here as well.

**Comment by dustymc** [comment]
This needs distilled down to a code table request or closed.

**Comment by AJLinn** [comment]
This issue should be referenced in the feature request - Add remarks to table collector https://github.com/ArctosDB/arctos/issues/8032

**Comment by dustymc** [comment]
I think this is all resolved, tentatively closing
